deltacat/tests/compute/resource_estimation/test_delta.py

```diff
@@ -21,21 +20,6 @@
 """
 
 
-@pytest.fixture(scope="function")
-def local_deltacat_storage_kwargs():
-    DATABASE_FILE_PATH_KEY, DATABASE_FILE_PATH_VALUE = (
-        "db_file_path",
-        "deltacat/tests/local_deltacat_storage/db_test.sqlite",
-    )
-    # see deltacat/tests/local_deltacat_storage/README.md for documentation
-    kwargs_for_local_deltacat_storage = {
-        DATABASE_FILE_PATH_KEY: DATABASE_FILE_PATH_VALUE,
-    }
-    yield kwargs_for_local_deltacat_storage
-    if os.path.exists(DATABASE_FILE_PATH_VALUE):
-        os.remove(DATABASE_FILE_PATH_VALUE)
-
-
```

Comment:
info: I assume the reason to remove those method is because they are moved to `conftest.py` file, however I don't see the `conftest/py` version of this method getting imported anywhere. How is the importing configured?

Reply:
got the answer by searching `conftest.py`.